src/stores/client.rs

```diff
@@ -60,15 +60,17 @@ impl ClientStore for sqlx::PgPool {
         let mut notification_query_builder =
             sqlx::QueryBuilder::new("DELETE FROM public.notifications WHERE client_id = ");
         notification_query_builder.push_bind(id);
-        notification_query_builder.push("and tenant_id = ");
+        notification_query_builder.push(" and tenant_id = ");
         notification_query_builder.push_bind(tenant_id);
         let notification_query = notification_query_builder.build();
 
         self.execute(notification_query).await?;
 
+        println!("Deleting client {}", id);
```

Comment:
Please remove debug lines or use `tracing`

Reply:
ups. thank you!